### DISCUSSION TITLE
minimum reader and writer version override

### DISCUSSION THREAD

**Discussion by washcycle** [delta-io/delta-rs#1049] [Q&A] [answered]
I have delta tables that use a higher minimum reader and writer version.

Is there a way to override the default limits or is that a bad idea?

`DeltaTableProtocolError: The table's minimum reader version is 2 but deltalake only supports up to version 1.`

Thanks!

**Comment by wjones127** [comment, marked as answer]
There is not; this error means we cannot read the table correctly. Trying to read it would give you bad data.

**Reply by washcycle** [reply]
I figured as much, thanks!
